web/admin/src/components/organisms/TheCoordinatorCreateForm.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <form @submit.prevent="handleSubmit">
+    <v-card>
+      <v-card-text>
+        <v-text-field
+          v-model="v$.companyName.$model"
+          :error-messages="getErrorMessage(v$.companyName.$errors)"
+          label="会社名"
+        />
+        <v-text-field
+          v-model="v$.storeName.$model"
+          :error-messages="getErrorMessage(v$.storeName.$errors)"
+          label="店舗名"
+        />
+        <div class="mb-2 d-flex">
+          <the-profile-select-form
+            class="mr-4 flex-grow-1 flex-shrink-1"
+            :img-url="formData.thumbnailUrl"
+            :error="thumbnailUploadStatus.error"
+            :message="thumbnailUploadStatus.message"
+            @update:file="updateThumbnailFileHandler"
+          />
+          <the-header-select-form
+            class="flex-grow-1 flex-shrink-1"
+            :img-url="formData.headerUrl"
+            :error="headerUploadStatus.error"
+            :message="headerUploadStatus.message"
+            @update:file="updateHeaderFileHandler"
+          />
+        </div>
+
+        <div class="d-flex">
+          <v-text-field
+            v-model="v$.lastname.$model"
+            :error-messages="getErrorMessage(v$.lastname.$errors)"
+            class="mr-4"
+            label="コーディネーター:姓"
```

Comment:
もろもろ `コーディネーター` -> `コーディネータ` の修正おなしゃす🙏

```suggestion
            label="コーディネータ:姓"
```